jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTest.java

```diff
@@ -690,10 +689,9 @@ public void onComplete(Result result)
 
     @ParameterizedTest
     @ArgumentsSource(ScenarioProvider.class)
-    @DisabledIfSystemProperty(named = "env", matches = "ci") // TODO: SLOW, needs review
     public void testRequestIdleTimeout(Scenario scenario) throws Exception
     {
-        long idleTimeout = 1000;
+        long idleTimeout = 100;
```

Comment:
Ah yes. This one I forgot to rework it, but since it's almost identical to the FCGI one below, let's only discuss that other one and I'll make them both work the same way.